docs/development/setup_development.rst

```diff
@@ -106,10 +108,10 @@ submissions are present. The test users have the following credentials:
 * **Password:** ``WEjwn8ZyczDhQSK24YKM8C9a``
 * **TOTP secret:** ``JHCO GO7V CER3 EJ4L``
 
-.. note:: The password and TOTP secret are the same for both accounts for
-   convenience during development.
+.. tip:: We're hosting a demonstration instance of SecureDrop so that anyone can see how   
+    it works. The demo site generates Time-based One-time Password (TOTP) that you can use with the password above. The password and TOTP secret are the same for both accounts for convenience during development. To get a generated token, go to: http://demo.securedrop.club/
```

Comment:
Nit: Let's move "The password and TOTP secret are the same for both accounts for convenience during development." out of the tip and place it above under the Username, Password, TOTP secret for both accounts. This is just so that there is not confusion as `dellsberg` is not a valid account on the demo iirc.